app/models/marc_fields/linked_serials.rb

```diff
@@ -0,0 +1,125 @@
+class LinkedSerials < MarcField
+  def values
+    relevant_fields.map do |field|
+      {
+        label: subfield_label(field),
+        values: field.subfields.map do |subfield|
+                  process_subfield(field, subfield)
+                end.flatten.compact
+      }
+    end
+  end
+
+  def to_partial_path
+    'marc_fields/linked_serials'
+  end
+
+  private
+
+  def process_subfield(field, subfield)
+    return { text: subfield.value } unless respond_to?("#{subfield.code}_subfield", true)
+    send("#{subfield.code}_subfield", field, subfield.value)
+  end
+
+  def a_subfield(field, value)
```

Comment:
I'm going to leave this as-is.  For better-or-worse, this is consistent with a pattern that exists in other marc parsing classes.